lib/resources/r4/encounter.yaml

```diff
@@ -547,6 +547,27 @@ fields:
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
 
+- name: Custom Attribute Value Extension
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute?_format=json
+  terminology_name: extension[x].extension[x].valueCodeableConcept
+  type: Extension
+  description: The value of the client defined custom attribute. This extension is nested under a <code>custom-attribute</code> extension.
+  action: terminology
+  binding:
+    description: The value of the client defined custom attribute. This binding is used for the <code>custom-attribute-value</code> extension nested under <code>custom-attribute</code> extensions.
+    terminology:
+      - display: Custom Attribute Code Value
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/&lt;code set&gt;
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#proprietary-codes
+      - display: Custom Attribute Nomenclature Value
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/nomenclature
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#proprietary-codes
```

Comment:
@MadhurRaj 
For Nomenclature value info_link, should this be the link we have now, or should it be https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#nomenclature ?

Reply:
yea, that sounds good.

Reply:
[Updated](https://github.com/cerner/fhir.cerner.com/pull/438/commits/c265e3a9724a67877c9cbd599f8a5800ed6223ac)